web/src/views/DandisetLandingView/DownloadDialog.vue

```diff
@@ -1,4 +1,8 @@
 <template>
+  <!--
+    Do we want to set the width here? Default is nearly the same.
+    If we do want to set wouldn't width=500 be sufficient?
+  -->
```

Comment:
This comment is in regards to the the following `v-menu`. Just wanted to call attention to it on whether it should be changed or not. No need for it to be done in this PR or at all.

Reply:
Why would we want to change this? If there's a problem with the width for this other menu, I would file an issue and/or open a different PR to discuss/deal with it.

Reply:
explained below.